ports/nrf/boards/TG-Watch02A/README.md

```diff
@@ -0,0 +1,201 @@
+# Setup
+
+The `feather52840` board is currently based on the `PCA10056` development
```

Comment:
No need to add this README since it is about a different board. (So, please delete it.)